### PR TITLE
Issue #1064 Fixed the echo add-on command to print empty line

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@hferentschik Agree, will add test.

File: pkg/minishift/addon/command/echo_command.go [hunk 1]
```diff
@@ -34,11 +34,19 @@ func NewEchoCommand(command string) *EchoCommand {
...
+	var err error
 
-	_, err := fmt.Print("\n  " + echo)
+	// split off the actual 'echo' command. As we need to print rest of the string
+	echoString := strings.Replace(c.rawCommand, "echo", "", 1)
+
+	//Remove leading and trailing spaces
+	echoString = strings.TrimSpace(echoString)
+	if echoString == "" {
+		_, err = fmt.Print("\n")
```
Comment (praveenkumar) [inline pkg/minishift/addon/command/echo_command.go]: Why can't we just use `fmt.Println()` which auto append newline?

Comment (LalatenduMohanty) [inline pkg/minishift/addon/command/echo_command.go]: @praveenkumar I think it is almost same thing. I used `Print("\n")` for consistency in the existing code and it is explicit in a way.

Comment (hferentschik) [inline pkg/minishift/addon/command/echo_command.go]: Right, it is more explicit. I definitely get it. It is like _stressing_ a word.

**Comment by hferentschik** [comment]
Merged, thanks.

**Pull request merged**